packages/react-components/react-slider/src/components/Slider/SliderBestPractices.md

```diff
@@ -0,0 +1,17 @@
+<details>
```

Comment:
Copied from existing docs (https://developer.microsoft.com/en-us/fluentui#/controls/web/slider). More or less makes sense to me, opened for suggestions.